src/openrct2/util/Util.h

```diff
@@ -11,6 +11,7 @@
 
 #include "../core/Money.hpp"
 
+#include <algorithm>
```

Comment:
This is a relatively heavy header that we deliberately tried to avoid in this common header.